trackers/class-wp-sentry-js-tracker.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+require_once __DIR__ . '/class-wp-sentry-tracker-base.php';
+
+/**
+ * WordPress Sentry Javascript Tracker.
+ */
+final class WP_Sentry_Js_Tracker extends WP_Sentry_Tracker_Base {
+
+	/**
+	 * Get the sentry tracker instance.
+	 *
+	 * @return WP_Sentry_Js_Tracker
+	 */
+	public static function get_instance() {
+		static $instance = null;
+		return $instance ?: $instance = new self( WP_SENTRY_PUBLIC_DSN );
```

Comment:
This is true, however since this is information only interesting for devs, I would like to keep the config in the `wp-config`. But it might not be a bad idea to somehow not use the constants in these classes, I agree. Going to think about that...